### ISSUE TITLE
3025U broadwell Acer Chromebook 15 intel drivers

### ISSUE THREAD

**Issue by raidencup** [dnschneid/crouton#1915]
Hello, there seems to be that mouse graphical glitch thing and possibly more stuff going on with the Acer Chromebook 15. I already tried updating the drivers multiple times, and they always break the chroot. Someone on Reddit mentioned that the drivers might be only compatible with the i5 and i3 Acer Chromebook 15. Others have also reported that updating the intel drivers also breaks their 3025U Chromebook. Is there anything we can do about it? Thanks! Btw Crouton is a Chromebook-saver! Thank you so much for your work.

I am also having graphical glitches while the cursor is not corrupt. It is fixed only by going to chromeos and then back to linux again.

**Comment by raidencup** [comment]
Okay this seems to be a known problem with Ubuntu and possibly its kernel support for broadwell. https://bugs.launchpad.net/xserver-xorg-video-intel/+bug/1432194

 Anyway, I just messed around purged then installed xorg again. Also installed the new test packages  at post number 100 then purged them because of dependencies. I seriously could not tell you how to replicate what I did but I can now report that everything is PERFECT. Not even the cursor glitch! Cheers.
